Packs/Gatewatcher/Integrations/Gcenterv102/Gcenterv102.yml

```diff
@@ -0,0 +1,445 @@
+category: Network Security
+commonfields:
+  id: Gcenterv102
+  version: -1
+configuration:
+- display: GCenter IP address
+  name: ip
+  type: 0
+  required: true
+- display: GCenter API token
+  name: token
+  type: 0
+  required: false
+- display: GCenter username
+  displaypassword: GCenter password
+  name: credentials
+  type: 9
+  required: false
+- display: Check the TLS certificate
+  name: check_cert
+  type: 8
+  required: false
+description: This is the Gatewatcher integration for getting started.
+display: GCenter v2.5.3.102
+name: Gcenterv102
+script:
+  commands:
+  - name: gw-list-alerts
+    description: List all alerts
+  - name: gw-get-alert
+    description: Get an alert by it's uid
+    arguments:
+    - name: uid
+      required: true
+      description: Alert identifier
+  - name: gw-es-query
+    description: Get Elasticsearch data
+    arguments:
+    - name: index
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - suricata
+      - malware
+      - codebreaker
+      - netdata
+      - syslog
+      description: Index to be queried
+      defaultValue: suricata
+    - name: query
+      required: true
+      description: Elaticsearch query
+      defaultValue: '{}'
+  - name: gw-add-malcore-list-entry
+    description: Add malcore whitelist/blacklist entry
+    arguments:
+    - name: type
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - white
+      - black
+      description: List type
+    - name: sha256
+      required: true
+      description: SHA256 to be added
+    - name: comment
+      required: false
+      description: Comment to be added
+    - name: threat
+      required: false
+      description: Comment to be added
+    outputs:
+    - contextPath: GCenter.Malcore.sha256
+      description: ''
+      type: String
+    - contextPath: GCenter.Malcore.created
+      description: ''
+      type: Date
+    - contextPath: GCenter.Malcore.comment
+      description: ''
+      type: String
+    - contextPath: GCenter.Malcore.threat
+      description: ''
+      type: String
+  - name: gw-del-malcore-list-entry
```

Comment:
I recommend you add descriptions for the different outputs.

Reply:
done